### PR TITLE
code refactoring

### PR DESCRIPTION
**Pull request by prayans**
1. Replace date-ui/date-fns with @date-io/moment
2. Export everything that is need to be exposed in the lib/index.tsx
   like export * fom 'filename'
3. Remove the default props while attaching fields.
4. Remove line import { IMUIDropDownTimePickerProps, MUIDropDownTimePickerProps } from 'react-forms/dist/lib/ml-form-builder/lib/MUIDropDownTimePicker'; as it is to be removed from the react-forms
5. While setting the date to the formikProps change to to the toISOString. [e.g.: new Date().toISOString()]
6. Remove all the unnecessary packages from package.json

### CODE REVIEW

File: src/lib/MUIDropDownTimePicker.tsx [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
+    formHelperTextProps?: FormHelperTextProps
+    startTime?: string | Date
+    endTime?: string | Date
+    interval?: number
+    amPm?: boolean
+    emptyMenuItemProps?: object
+    menuItemProps?: object
+    inputLabelProps?: object
+}
+export interface MUIDropDownTimePickerProps extends IFieldProps {
```
Comment (sirajalam049) [inline src/lib/MUIDropDownTimePicker.tsx]: Change MUIDropDownTimePickerProps to DropDownTimePickerProps

File: src/lib/MUIDropDownTimePicker.tsx [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
 import React, { FC } from 'react';
 import moment from 'moment'
-import { FormConfig } from 'react-forms';
-import { IMUIDropDownTimePickerProps, MUIDropDownTimePickerProps } from 'react-forms/dist/lib/ml-form-builder/lib/MUIDropDownTimePicker';
+import { FormConfig, IFieldProps } from 'react-forms';
+// import { IMUIDropDownTimePickerProps, MUIDropDownTimePickerProps } from 'react-forms/dist/lib/ml-form-builder/lib/MUIDropDownTimePicker';
+// import { getFieldError, MenuOptionObject } from 'react-forms/dist/lib/ml-form-builder/Utils';
 import { getFieldError, MenuOptionObject } from './Utils';
 
+export interface IMUIDropDownTimePickerProps extends SelectProps {
```
Comment (sirajalam049) [inline src/lib/MUIDropDownTimePicker.tsx]: Change MUIDropDownTimePickerProps to DropDownTimePickerFieldProps

File: src/lib/index.tsx [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
 import React from 'react';
 import { MUIDatePicker, MUITimePicker } from './MUIDateTimePicker'
-export { MUIDatePicker, MUITimePicker } from './MUIDateTimePicker'
+export * from './MUIDateTimePicker'
 import { MUIDropDownTimePicker } from './MUIDropDownTimePicker'
-export { MUIDropDownTimePicker } from './MUIDropDownTimePicker'
-import '@date-io/date-fns'
+export * from './MUIDropDownTimePicker'
+import '@date-io/moment'
+// "date-fns": "^2.16.1",^1.3.13
```
Comment (sirajalam049) [inline src/lib/index.tsx]: This comment is unnecessary, we can remove it.

File: src/lib/index.tsx [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
 
-attachField('date-picker-new', <MUIDatePicker />, { variant: 'inline', label: 'Select Date' });
-attachField('time-picker-new', <MUITimePicker />, { variant: 'inline', label: 'Select Time' });
+attachField('date-picker-new', <MUIDatePicker />);
```
Comment (sirajalam049) [inline src/lib/index.tsx]: change date-picker-new to  mui-date-picker

Make similar changes to the other attachFields too.

**Review by sirajalam049 — CHANGES_REQUESTED** [review]

File: src/lib/MUIDropDownTimePicker.tsx (hunk 1 above)
Comment (prayans) [reply src/lib/MUIDropDownTimePicker.tsx]: @sirajalam049  Done the requested changes.

File: src/lib/MUIDropDownTimePicker.tsx (hunk 2 above)
Comment (prayans) [reply src/lib/MUIDropDownTimePicker.tsx]: @sirajalam049 Done the requested changes.

File: src/lib/index.tsx (hunk 3 above)
Comment (prayans) [reply src/lib/index.tsx]: @sirajalam049 Done the requested changes.

File: src/lib/index.tsx (hunk 4 above)
Comment (prayans) [reply src/lib/index.tsx]: @sirajalam049 Done the requested changes.

**Review by sirajalam049 — APPROVED** [review]
Thank You!

**Pull request merged**